python/setup.py

```diff
@@ -305,7 +305,7 @@ def get_packages(self):
         # Light weight requirement, can be replaced with "typing" once
         # we deprecate Python 3.7 (this will take a while).
         "typing_extensions; python_version < '3.8'",
-        "virtualenv",  # For pip runtime env.
+        "virtualenv>=20.0.24",  # For pip runtime env.
```

Comment:
Thanks! Can you also change it in `requirements/requirements.txt`?

Reply:
sure!